UITests/TrackingProtectionTests.swift

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Storage
+import EarlGrey
+@testable import Client
+
+class TrackingProtectionTests: KIFTestCase {
+    
+    private var webRoot: String!
+        
+    override func setUp() {
+        super.setUp()
+        webRoot = SimplePageServer.start()
+        BrowserUtils.dismissFirstRunUI()
+    }
+    
+    override func tearDown() {
+        BrowserUtils.clearPrivateData(tester: tester())
```

Comment:
Idea for future: I wonder if this adds a lot of time to tests, we might be able to speed up tests by doing:
`static func tearDown() {  /* do heavy cleanup after all tests in this file ran rather than after each test */}`

This isn't related to this file, I am just mentioning because at some point it would be nice to look at speeding up the tests.

Reply:
will look into it, I think some tests actually require cleanup between tests in the same file